assignments/#10.1.py

```diff
@@ -0,0 +1,34 @@
+"""
+This program split given large number into hundreds, tens and units.
+"""
+
+
+def split_number(num):
+    result = []
+    i = 10
+    while num > 0:
+        c = num % i
+        num = num - c
+        i = i*10
+        result.append(c)
+    return list(reversed(result))
+
+
+if __name__ == '__main__':
+    print(__doc__)
+
+    test_cases = [
+        (1, [1]),
+        (1025, [1000,0,20,5]),
+    ]
+
+    for testnum, test in enumerate(test_cases):
+        inp = test[0]

```

Comment:
In this loop we call the same function for every test case we made before. 
Add some, to get a feeling of what the testing your code is. The nice thing about it: every time you change your function, you just relaunch the whole testing process and see is this function still correct.